jest-config.js

```diff
@@ -2,13 +2,9 @@ module.exports = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
   moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
-  testMatch: ["**/jest/**/?(*.)+(test).[jt]s?(x)"],
-  testPathIgnorePatterns: ["victory-native"],
+  testPathIgnorePatterns: ["victory-native", "node_modules", "config"],
   transform: {
-    "^.+/test/.+\\.(ts|tsx|js|jsx)$": [
-      "babel-jest",
-      { configFile: "./.babelrc.js" }
-    ]
+    "\\.[jt]sx?$": ["babel-jest", { configFile: "./.babelrc.js" }]
```

Comment:
Oh, it looks like we're _re_-transpiling `packages/victory-*/lib/**/*.js` that was already transpiled here.